src/test/java/org/folio/rest/impl/MockServer.java

```diff
@@ -260,6 +285,80 @@ private Router defineRoutes() {
     return router;
   }
 
+
+  private void handleGetAcquisitionsMemberships(RoutingContext ctx) {
+    logger.info("handleGetAcquisitionsMemberships got: " + ctx.request().path());
+
+    String query = StringUtils.trimToEmpty(ctx.request().getParam("query"));
+    if (query.contains(BAD_QUERY)) {
+      serverResponse(ctx, 400, APPLICATION_JSON, Response.Status.BAD_REQUEST.getReasonPhrase());
+    } else {
+
+      Matcher userIdMatcher = Pattern.compile(".*userId==(\\S+).*").matcher(query);
+      final String userId = userIdMatcher.find() ? userIdMatcher.group(1) : EMPTY;
+
+      AcquisitionsUnitMembershipCollection memberships;
+      try {
+        memberships = new JsonObject(ApiTestBase.getMockData(ACQUISITIONS_MEMBERSHIPS_COLLECTION)).mapTo(AcquisitionsUnitMembershipCollection.class);
+      } catch (IOException e) {
+        memberships = new AcquisitionsUnitMembershipCollection();
+      }
+
+      if (StringUtils.isNotEmpty(userId)) {
+        memberships.getAcquisitionsUnitMemberships().removeIf(membership -> !membership.getUserId().equals(userId));
+        List<String> acquisitionsUnitIds = extractIdsFromQuery(ACQUISITIONS_UNIT_ID, query);
+        if (!acquisitionsUnitIds.isEmpty()) {
+          memberships.getAcquisitionsUnitMemberships().removeIf(membership -> !acquisitionsUnitIds.contains(membership.getAcquisitionsUnitId()));
+        }
+      }
+
+      JsonObject data = JsonObject.mapFrom(memberships.withTotalRecords(memberships.getAcquisitionsUnitMemberships().size()));
+      addServerRqRsData(HttpMethod.GET, ACQUISITIONS_MEMBERSHIPS, data);
+      serverResponse(ctx, 200, APPLICATION_JSON, data.encodePrettily());
+    }
+  }
+
+  private void handleGetAcquisitionsUnits(RoutingContext ctx) {
+    logger.info("handleGetAcquisitionsUnits got: " + ctx.request().path());
+    String tenant = ctx.request().getHeader(OKAPI_HEADER_TENANT);
+    String query = StringUtils.trimToEmpty(ctx.request().getParam("query"));
+
+    AcquisitionsUnitCollection units;
+
+    try {
+      if (PROTECTED_READ_ONLY_TENANT.equals(tenant)) {
+        units = new AcquisitionsUnitCollection();
+      } else {
+        units = new JsonObject(ApiTestBase.getMockData(ACQUISITIONS_UNITS_COLLECTION)).mapTo(AcquisitionsUnitCollection.class);
+      }
+    } catch (IOException e) {
+      units = new AcquisitionsUnitCollection();
+    }
+
+    if (query.contains(BAD_QUERY)) {
+      serverResponse(ctx, 400, APPLICATION_JSON, Response.Status.BAD_REQUEST.getReasonPhrase());
+    } else {
+
+      if(query.contains("name==")) {
```

Comment:
Should this `name==` if() be removed because it's not being used?

Reply:
Removed